draft-ietf-quic-multipath.md

```diff
@@ -1414,23 +1414,22 @@ MAX_PATHS Frame {
 MAX_PATHS frames contain the following field:
 
 Maximum Path Identifier:
-: The latest Path ID that can be used for this connection.
+: The largest Path ID that can be used for this connection.
   This value cannot exceed 2^32-1, as it is not
-  possible to encode Path IDs larger than 2^32-1.
+  possible to encode Path IDs larger than 2^32-1 as variable integer.
 
```

Comment:
Uh, no. We could encode up to 2^62 - 1. The real reason is that we cannot encode more than 32 bit of path-ID and 64 bit of packet number in the 96 bit AEAD nonce.

Reply:
That doesn't make sense. In this case we would allow 2^62 - 1 path IDs but require a key update earlier.

Reply:
I would be fine with that. It was actually the original design. But there are complexities. For example, can you have both a long-lived path number 1 and then create path ID 2^32 + 1?

Reply:
hm, okay I guess 2^32 - 1 paths are usually enough... In this case I think we need to actually use a MUST here and we should maybe remove the respective text from the intro in order to define it only once.

It's late here; I will update the PR accordingly tomorrow which gives also others a chance to comment.

Reply:
Also based on @huitema text below we need to add this:

```suggestion

The Maximum Path Identifier MUST NOT be lower than the value
advertised in the initial_max_paths transport parameter. Receipt
of a Path ID lower than the value of initial_max_paths transport parameter
MUST be treated as a connection error of type MP_PROTOCOL_ERROR.
```